Ambiguous-Squids/Bangazon_initial_site
# As login

# Description
Base template, login/logout views, side nav and user is able to see authenticated views

# Related Ticket(s)
#26 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

## Documentation

[ ] There is new documentation in this pull request that must be reviewed..

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout <feature_branch>
```

## Steps to Test
Outline the steps to test the code in this pull request.

1. Fire up server
2. See if user can login
